.github/workflows/push.yml

```diff
@@ -674,7 +673,28 @@ jobs:
             }
           }
           BODY
-
+      - uses: actions/checkout@v3
```

Comment:
Not needed action.

Reply:
Fixed in [662da9f](https://github.com/island-is/island.is/pull/7703/commits/662da9fe325ce0d2b32ee1c69694d1ea86921339)